CHANGELOG.md

```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.0] Preview Release
+### Added
+- Added support in SQLServerBulkCopy to allow Pooled/XA Connection instances during object creation [#968](https://github.com/Microsoft/mssql-jdbc/pull/968)
+- Added SQLServerParameterMetadata improvements with improved code coverage [#973](https://github.com/Microsoft/mssql-jdbc/pull/973)
```

Comment:
Move to Changes as "Improved SQLServerParameterMetadata API implementations and code coverage"